### ISSUE TITLE
HTML transitions not working smoothly

### ISSUE THREAD

**Issue by Xeddar** [RPTools/maptool#2725]
**Describe the bug**
Transitions on html elements don't appear to be rendering properly. For example, when updating the position of elements through javascript on an element that has a transition in its css, the element often doesn't move, gets stuck, or snaps suddenly to its new position. 

It seems that causing the overlay to re-render something (such as hovering over a button or clicking a link) will refresh the transition's animation.

**To Reproduce**
1. Run the attached macro in 1.9.0 - beta 2
2. Click on the "Move the box..." link 1+ times
3. See the box fail to transition from side to side smoothly

**Expected behavior**
The transitions should move smoothly as expected

**MapTool Info**
- Version: 1.9.0 - beta 2
- Working as expected in 1.8.3
- 1.8.4/1.8.5 not tested

**Desktop (please complete the following information):**
 - Windows 10

[BrokenTransitionDemonstraton.zip](https://github.com/RPTools/maptool/files/6631582/BrokenTransitionDemonstraton.zip)

**Comment by cwisniew** [comment]
Tested on Mac OS, same issue occurs

**Comment by Phergus** [comment]
Confirmed working on 1.8.5 and with noted behavior in current dev branch.

**Comment by BlackwingNL** [comment]
Confirmed that in 1.9.2 the problem also exists in frames and dialogs made with `frame5()` and `dialog5()`

**Comment by aliasmask** [comment]
I'm told my issue could be related.
``'
There seems to be a behavior change in CSS in MT 1.9.2 (after 1.8.5). When using radio buttons to show tabbed content (it only shows content for current radio button selected) it doesn't load the content until I mouseout of radio button selection rather than when selected. You can see the behavior by running **Test Tab Menu**. It has something to do with the fadeIn animation.
```
[lib_players 5-12-21.rptok.zip](https://github.com/RPTools/maptool/files/6760440/lib_players.5-12-21.rptok.zip)

Originally posted in discord: https://discord.com/channels/296230822262865920/296657960720007169/861305472384761878

**Comment by FullBleed** [comment]
I ran into this using a CSS text effect/animation.  I had various text in the frame with the effect and if I kept the mouse moving over all of the various pieces of text it would kind of keep the animation going, but when I stopped it would stop.

**Comment by aliasmask** [comment]
This is very restrictive on having some nice transitions for html forms. Any idea why it's still broken and how to fix? It used to work properly in 1.8.5. I just tested it again in 1.11.0 beta 5 and still has same issue.

**Comment by cwisniew** [comment]
> This is very restrictive on having some nice transitions for html forms. Any idea why it's still broken and how to fix? It used to work properly in 1.8.5. I just tested it again in 1.11.0 beta 5 and still has same issue.

I dont think it ever worked correctly on Mac OS. If it worked in Windows for 1.8.5 but not since 1.9 its probably down to JavaFX version. We will be upgrading Java for 1.12 and the latest JavaFX at that time. This may fix the problem... The downside of course is this may break again the Java/JavaFX upgrade after that.

**Comment by cwisniew** [comment]
This can't be fixed with the way things currently stand see #3419 for more context.
